### PR TITLE
Added Ubiquiti

### PR DESCRIPTION
**Pull request by marcolino7**
Added Ubiquiti

### CODE REVIEW

**Comment by Rygle** [comment]
Icons in the prescribed format.

ubiquiti 256x256 tinypng
ubiquiti!

**Review by mvdkleijn — CHANGES_REQUESTED** [review]
Icon file is missing. An icon file in either SVG or PNG format is required.
Please see:

- https://github.com/linuxserver/Heimdall-Apps/wiki/Submitting-an-app
- https://github.com/linuxserver/Heimdall-Apps/wiki/Image-requirements

File: Ubiquiti/app.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "847518631182ef7bf3b4a8c6ec4bd1e602750726",
+    "name": "Ubiquiti",
+    "website": "NA",
```
Comment (mvdkleijn) [inline Ubiquiti/app.json:4]: This should probably be something like: https://ui.com

File: Ubiquiti/app.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    "license": "BSD Zero Clause License",
```
Comment (mvdkleijn) [inline Ubiquiti/app.json:5]: Ubiquiti is BSD?!?

File: Ubiquiti/app.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "847518631182ef7bf3b4a8c6ec4bd1e602750726",
+    "name": "Ubiquiti",
```
Comment (mvdkleijn) [inline Ubiquiti/app.json:3]: Is this entry for their Unifi UI? Maybe a more specific name would be good.

File: Ubiquiti/app.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    "website": "NA",
+    "license": "BSD Zero Clause License",
+    "description": "Need to update the \"Unifi\" listing to the modern Ubiquiti logo and change to Dark to make it look nicer :)",
```
Comment (mvdkleijn) [inline Ubiquiti/app.json:6]: Please describe the product / app.

**Comment by mvdkleijn** [comment]
Please note that there is already a Unifi app, so not sure why you're trying to add a "Ubiquiti" one.

**Comment by mvdkleijn** [comment]
- no icon
- metadata issues
- linting issues
- no reply from PR owner

closing to cleanup, feel free to comment / reopen when desired

**Pull request closed without merging**